### DISCUSSION TITLE
Need encryption for datasource password

### DISCUSSION THREAD

**Discussion by vishwa-ims** [RevealBi/Reveal.Sdk#38] [ASP.NET Core]
Hi 
We are planning to store encrypted password for datasources . But I think that will impact datasource connection because you are directly picking datasource connection details from my database table. Could you please help me to know how I can get it implemented ?

**Comment by vishwa-ims** [comment]
@brianlagunas  Any updates please ?

**Reply by brianlagunas** [reply]
Your question is too vague. I do not understand what you are asking exactly. This doesn't seem like it is related to the Reveal SDK specifically, but rather how you are handling your encrypted passwords.
